src/services/expenses.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const { Op } = require('sequelize');
+const { generateId } = require('../functions/generateId');
+const { Expense } = require('../models/expenses');
+
+const getAll = async() => {
+  const allExpenses = await Expense.findAll();
+
+  return allExpenses;
+};
+
+const getById = async(expenseId) => {
+  return Expense.findByPk(expenseId);
+};
+
+const getByParameters = async(
+  userId = null,
+  categories = null,
+  from = new Date(0),
+  to = new Date(),
+) => {
+  const where = {
+    spentAt: {
+      [Op.between]: [from, to],
+    },
+  };
+
+  if (userId) {
+    where.userId = +userId;
+  }
+
+  if (categories !== null) {
+    where.category = {
+      [Op.in]: categories,
+    };
+  } else {
+    where.category = {
+      [Op.not]: null,
+    };
+  }
+
+  const foundExpenses = await Expense.findAll({ where });
+
+  return foundExpenses;
+};
+
+const create = (data) => {
+  const id = generateId();
```

Comment:
Instead of generating id in code, you can specify `autoIncrement: true` in your id column options in model. [Docs](https://sequelize.org/docs/v6/core-concepts/model-basics/#column-options). This way, it will create new ids by itself